class-11-spa-routing/pair-assignment/starter-code/scripts/adminController.js

```diff
@@ -0,0 +1,13 @@
+(function(module) {
+  var adminController = {};
+
+  Article.fetchAll(articleView.initAdminPage);
```

Comment:
we need to do this any time we hit this route, so this call should be inside adminController.index